backend/src/server.js

```diff
@@ -1,41 +1,51 @@
 import { ApolloServer } from 'apollo-server'
 
-import { makeExecutableSchema } from '@graphql-tools/schema'
 import { loadTypedefsSync } from '@graphql-tools/load'
 import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader'
 
+import { makeAugmentedSchema } from 'neo4j-graphql-js'
+import neo4jDriver from './neo4j-driver'
+
 import { applyMiddleware } from 'graphql-middleware'
 
-import { InMemoryNewsDS } from './db'
-import resolvers from './resolvers'
+import Resolvers from './resolvers'
 import permissions from './authorisation'
+import * as path from 'path'
+import { NeodeDS } from './NeodeDS'
+import { stitchSchemas } from '@graphql-tools/stitch'
+import * as fs from 'fs'
 
-const db = new InMemoryNewsDS()
-// db.createUser('Jonas')
-// db.createUser('Michelle')
-// db.createPost('This is a post!', 42, 'Jonas')
-// db.createPost('This is a another post!', 1337, 'Jonas')
-// db.createPost('This is a yet another post!', 1234, 'Michelle')
+const db = new NeodeDS()
 
 export default class NewsServer {
   constructor (opts) {
+    const dbSchema = makeAugmentedSchema({
+      typeDefs: fs.readFileSync(path.join(__dirname, 'nodes.graphql'), 'utf-8')
+    })
+
+    const typeDefs = loadTypedefsSync(path.join(__dirname, 'api.graphql'), {
+      loaders: [
+        new GraphQLFileLoader()
+      ]
+    }).map(source => source.document)
+
+    const resolvers = Resolvers({ subschema: dbSchema })
+
     const defaults = {
       schema:
         applyMiddleware(
-          makeExecutableSchema({
-            typeDefs: loadTypedefsSync('src/typeDefs.graphql', {
-              loaders: [
-                new GraphQLFileLoader()
-              ]
-            }).map(source => source.document),
+          stitchSchemas({
+            subschemas: [dbSchema],
+            typeDefs,
             resolvers
           }),
           permissions
```

Comment:
Do they include to
`(⭐ You) deny the client to call mutations and queries of your subschema directly.
`?